tekton/ci/jobs/tekton-image-build.yaml

```diff
@@ -150,7 +150,7 @@ spec:
         params:
           - name: CONTEXT
             value:
-              - $(tasks.check-git-files-changed.results.files[0])
+              - $(tasks.check-git-files-changed.results.files)
```

Comment:
I thought this looks good, but I tried running it and the the variable is not resolved correctly:

```
[check-git-files-changed : check-files-changed] tekton/images/ko/Dockerfile
[check-git-files-changed : check-files-changed] tekton/images/kubectl/Dockerfile
[check-git-files-changed : check-files-changed] + CHECK=passed
[check-git-files-changed : check-files-changed] + printf passed

[images-build : build-and-push] No Dockerfile found in /workspace/source/$(tasks.check-git-files-changed.results.files)'s directory or any of its parents
```

The reason is that the version of Tekton running on the cluster is not recent enough :facepalm:
I will update the cluster as soon as possible, and let you know once it's done.